src/scandeval/dataset_tasks.py

```diff
@@ -110,6 +110,103 @@ def get_all_dataset_tasks() -> dict[str, DatasetTask]:
 )
 
 
+MULTIPLE_CHOICE = DatasetTask(
+    name="multiple-choice",
+    supertask="multiple-choice",
+    metrics=[  # Not sure
+        MetricConfig(
+            name="mcc",
+            pretty_name="Matthew's Correlation Coefficient",
+            huggingface_id="matthews_correlation",
+            results_key="matthews_correlation",
+        ),
+        MetricConfig(
+            name="macro_f1",
+            pretty_name="Macro-average F1-score",
+            huggingface_id="f1",
+            results_key="f1",
+            compute_kwargs=dict(average="macro"),
+        ),
+    ],
+    labels=[],  # Maybe add?
+)
+
+
+TEXT_TO_TEXT = DatasetTask(
+    name="text-to-text",
+    supertask="text-to-text",
+    metrics=[
+        MetricConfig(
+            name="bertscore",
+            pretty_name="BERTScore",
+            huggingface_id="bertscore",
+            results_key="f1",
+            postprocessing_fn=lambda raw_score: (raw_score, f"{raw_score:.2%}"),
+            compute_kwargs=dict(model_type="microsoft/mdeberta-v3-base", device="cpu"),
+        ),
+        MetricConfig(
+            name="rouge-1",
+            pretty_name="ROUGE-1",
+            huggingface_id="rouge",
+            results_key="rouge1",
+            postprocessing_fn=lambda raw_score: (raw_score, f"{raw_score:.2%}"),
+        ),
+        MetricConfig(
+            name="rouge-2",
+            pretty_name="ROUGE-2",
+            huggingface_id="rouge",
+            results_key="rouge2",
+            postprocessing_fn=lambda raw_score: (raw_score, f"{raw_score:.2%}"),
+        ),
+        MetricConfig(
+            name="rouge-l",
+            pretty_name="ROUGE-L",
+            huggingface_id="rouge",
+            results_key="rougeL",
+            postprocessing_fn=lambda raw_score: (raw_score, f"{raw_score:.2%}"),
+        ),
+        MetricConfig(
+            name="bleu",
+            pretty_name="BLEU",
+            huggingface_id="bleu",
+            results_key="bleu",
+            postprocessing_fn=lambda raw_score: (raw_score, f"{raw_score:.2%}"),
+        ),
+        MetricConfig(
+            name="chrf++",
+            pretty_name="chrF++",
+            huggingface_id="chrf",
+            results_key="score",
+            postprocessing_fn=lambda raw_score: (raw_score, f"{raw_score:.2f}%"),
+            compute_kwargs=dict(word_order=2),
+        ),
+        MetricConfig(
+            name="character",
+            pretty_name="CharacTER",
+            huggingface_id="character",
+            results_key="cer_score",
+            postprocessing_fn=lambda raw_score: (raw_score, f"{raw_score:.2%}"),
+        ),
+    ],
+    labels=[],
+)
+
+
+RAW_TEXT = DatasetTask(
```

Comment:
Yeah I didn't quite like it either. I'm hesitant at calling it `PERPLEXITY`, since that's just a metric, and we might include other metrics that measure "knowledge about the given text". `CAUSAL_LM` and `AUTOREGRESSIVE_LM` can maybe be misleading, since (I'm guessing) it should be possible benchmark seq2seq models on these tasks as well. `PREDICTIVE_LM` could maybe work, but it doesn't really seem to capture what we're trying to measure.

Maybe something like `RAW_COMPREHENSION`? Or maybe simply `KNOWLEDGE`, albeit that's a bit vague...

Reply:
Understandable. `RAW_COMPREHENSION` is a bit misleading are we measuring comprehension? (or knowedge) Maybe `TEXT_MODELLING`?

Reply:
I like `TEXT_MODELLING`! Let's say that.